site/src/comparison.rs

```diff
@@ -6,6 +6,7 @@ use crate::api;
 use crate::github;
 use crate::load::SiteCtxt;
 
+use collector::compare::DEFAULT_SIGNIFICANCE_THRESHOLD;
```

Comment:
Let's keep this in the site. It's unclear how to handle statistical computation in the comparison CLI. Because there we won't have almost any historical records, and probably the user wants to see all data.

So let's just keep the statistical stuff in `site` for now, and just show *all* the data in the collector CLI.